### ISSUE TITLE
When jumped, Ruler position not sticking to/moveing from m1 as expected

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#209]
@zepumph, just noting the movement of the ruler when using the keyboard is a little buggy.
Perhaps this is happening because I haven't given you general movement responses yet? In any case with or without general movement responses we will want to fix this behaviour.

1. There seems to be a rightward bound-border that is not respecting the most rightward possible position of center of m1.
2. I can only move the ruler beyond that rightward bound/border by using the jump keys: JC.
3. Further movement in any direction snaps the ruler left back to this "rightward bound-border".
4. This snap-left happens with arrow keys or WASD keys.
5. JC shortcut always works to get it back right beyond the bounds, but then it snaps back with any other movement.

**Can't move further right than this with keyboard:**
![Screen Shot 2019-11-25 at 12 54 19 PM](https://user-images.githubusercontent.com/1902116/69558896-95686280-0f83-11ea-9e7b-b0609533ea60.png)
**Can jump further right with JC shortcut**
![Screen Shot 2019-11-25 at 12 54 37 PM](https://user-images.githubusercontent.com/1902116/69558854-84b7ec80-0f83-11ea-884c-184944f9ea60.png)
**Snap left with pressing the up arrow**
The snap left happens with any attempt to move ruler.
![Screen Shot 2019-11-25 at 12 55 01 PM](https://user-images.githubusercontent.com/1902116/69558911-9b5e4380-0f83-11ea-967e-b734a5f507f8.png)

**Comment by zepumph** [comment]
Good catch! This is because the drag bounds for the ruler were not large enough for when the left mass was so far over to the right. The "jumping" bypassed those drag bounds, and so when dragging after a jump it was "snapped" back to within the drag bounds. I update the bounds to support the zero mark going all the way to the right object position. Please review.
